src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -445,13 +445,21 @@ public boolean isRecipeAlterationAllowed()
      */
     private boolean hasSpaceForMoreRecipes()
     {
+
+        return maxRecipes() > getRecipes().size() + 1;
+    }
+
+    /**
+     * Gets the maximum number of recipes a building may have at the current time.
+     */
+    private int maxRecipes() {
```

Comment:
the formatting is off here slightly.

Reply:
I see two things on formatting here: 
1) the curly brace needs to be on the next line
2) maxRecipes should be getMaxRecipes

Reply:
Changed the formatting, and the function name